docs/src/api/class-formdata.md

```diff
@@ -14,6 +14,77 @@ FormData form = FormData.create()
 page.request().post("http://localhost/submit", RequestOptions.create().setForm(form));
 ```
 
+## method: FormData.append
+* since: v1.43
+- returns: <[FormData]>
+
+Appends a new value onto an existing key inside a FormData object, or adds the key if it
+does not already exist. File values can be passed either as `Path` or as `FilePayload`.
+Multiple fields with the same name can be added.
+
+The difference between set() and append() is that if the specified key already exists,
```

Comment:
```suggestion
The difference between [`method: FormData.set`] and [`method: FormData.append`] is that if the specified key already exists,
```

Reply:
These links will not work in js and python.